### ISSUE TITLE
voice control on Google Home does not work

### ISSUE THREAD

**Issue by manjh** [andrei-tatar/node-red-contrib-smartnora#23]
I have installed Smart Nora and migrated my devices/
Everything shows up on the Google Home panel on my Android phone, and I can switch everything just fine by clicking the icons.
But when I try to control them with voice control, Google complains that the lamps have not bee set yet.
This could be a Google Home issue, but I cannot find any clue....

**Comment by andrei-tatar** [comment]
Hmm, that's weird. I haven't seen this before. So if you click the "mic" icon and ask something it says it's not setup? Or are you using a google home device?
![image](https://user-images.githubusercontent.com/6473183/116243963-2393da00-a770-11eb-80b7-bd27f206f7d6.png)

**Comment by mash87** [comment]
I have a slightly different problem. Having installed Smart NORA 1.0.4, I have voice control via my Galaxy S7 phone but not through my Google Home/Assistant device and a Lenovo Smart Clock with Google Assistant.
Both these devices are trying to connect to the NORA service even though NORA is not showing under Linked Services. Only Smart NORA is showing. The reply to a voice command is: 'Sorry, I couldn't reach NORA'. 
As NORA is missing from Linked Services list I am unable to unlink the account.

**Comment by andrei-tatar** [comment]
@mash87 is the same Google account setup in the Google Home devices as you have on your phone?

**Comment by mash87** [comment]
@andrei-tatar Thank you soooooooooo much, that's it, different accounts. NORA account now unlinked and everything works!

**Comment by manjh** [comment]
My Google Mini worked just fine, I was able to ask questions like time etcetera, it was just the switching of lights thru Nora that did not work.
The good news is that the problem has gone away. This morning I decided to try and reset the Mini to factory settings, and re-install. Now I can issue voice commands just like I used to before migrating to Smart Nora...
Happy that it now works, unhappy because the root cause will remain a mystery...
